db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_add_vm.yaml

```diff
@@ -9,4 +9,4 @@ object:
     description: 
   fields:
   - rel5:
-      value: "/System/event_handlers/event_action_refresh?target=src_ems_cluster"
+      value: "/System/event_handlers/event_action_refresh?target=src_vm"
```

Comment:
So instead of calling `event_action_refresh` here and having to do some special casing in the event catcher ([here](https://github.com/ManageIQ/manageiq/pull/10934/files#diff-9577813285f39dac068a42c586e9d7f1R45)) a better approach would be to create a new method like `event_action_new_vm`

You'll need a builtin automate method similar to [miq_event_action_refresh](https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/miq_ae_builtin_method.rb#L109) and an EmsEvent method similar to [refresh](https://github.com/ManageIQ/manageiq/blob/master/app/models/ems_event/automate.rb#L5) which puts a `new_vm` action on the queue for the `Refresh Worker`.  This way creating the VM record is done by the `Refresh Worker`